src/sections/Docker-Meshery/docker-extension-meshery.js

```diff
@@ -206,6 +229,42 @@ const DockerExtensionMeshery = () => {
                   name="org"
                   required
                 />
+                <div className="accounts">
+                  <label className="form-name">Account(s) to Connect<span className="required-sign">*</span></label>
+                  <p>
+                    Choose between Twitter, Google, LinkedIn, and GitHub, provide the username/handle of your user account for your preferred identity provider. Selected beta program participants will receive a free Meshery Cloud account and have full access MeshMap enabled for each of the following user accounts that you provide. Please provide at least one account.
+                  </p>
+                  {validateAccounts && <p style={{ margin: "0px", color: "red" }}>{errorAccounts}</p>}
+                  <div className="accounts_group">
+                    <label htmlFor="google" className="form-name">Google</label>
+                    <Field type="text" placeholder="my-address@gmail.com" className="text-field" id="google" name="google" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" />
+                    <label htmlFor="github" className="form-name">GitHub</label>
+                    <Field type="url" placeholder="https://github.com/" className="text-field" id="github" name="github" pattern="http(s?)(:\/\/)((www.)?)github.com(\/)([a-zA-z0-9\-_]+)" />
```

Comment:
The user needs to give only the username/handle. So, update the type of input to text.

Reply:
Same for below 2 fields